packages/sitecore-jss-tracking/src/dataModels.ts

```diff
@@ -33,3 +33,19 @@ export interface PageViewInstance {
   /** URL of page to track - should be absolute route path but can accept any value */
   url: string;
 }
+
+export interface PageViewData {
+  [key: string]: unknown;
```

Comment:
I think it makes sense to add a comment here as well.

Reply:
This line is removed